### PR TITLE
[RFR]Fix provision stack tests

### PR DESCRIPTION
**Pull request by niyazRedhat**
Purpose or Intent
=================

{{pytest: cfme/tests/services/test_provision_stack.py  -v}}

### CODE REVIEW

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

File: cfme/tests/services/test_provision_stack.py [hunk 1]
```diff
@@ -182,17 +182,17 @@ def test_provision_stack(order_stack):
...
 
     Metadata:
         test_flag: provision
 
     Polarion:
-        assignee: sshveta
+        assignee: nansari
         initialEstimate: 1/4h
         casecomponent: Services
+        tags: service
```
Comment (sshveta) [inline cfme/tests/services/test_provision_stack.py]: Tag will be stack here

File: cfme/tests/services/test_provision_stack.py [hunk 2]
```diff
@@ -218,21 +217,20 @@ def test_remove_non_read_only_orch_template(appliance, provider, template, servi
...
```
Comment (sshveta) [inline cfme/tests/services/test_provision_stack.py]: Tag will be stack here

File: cfme/tests/services/test_provision_stack.py [hunk 3]
```diff
@@ -243,23 +241,25 @@ def test_remove_read_only_orch_template_neg(appliance, provider, template, servi
...
 
     Metadata:
         test_flag: provision
 
     Polarion:
-        assignee: sshveta
+        assignee: nansari
         initialEstimate: 1/4h
         casecomponent: Services
+        tags: service
```
Comment (sshveta) [inline cfme/tests/services/test_provision_stack.py]: tags : stack

File: cfme/tests/services/test_provision_stack.py [hunk 4]
```diff
@@ -243,23 +241,25 @@ def test_remove_read_only_orch_template_neg(appliance, provider, template, servi
     Steps:
     1. Order Service which uses Orchestration template
     2. Try to remove this Orchestration template
+    3. Return false if button is disable
```
Comment (sshveta) [inline cfme/tests/services/test_provision_stack.py]: This should be "Check if remove item is disabled."

**Pull request merged**